save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -46,12 +46,12 @@ class FixPlugin(
         .build()
 
     @Suppress("TOO_LONG_FUNCTION")
-    override fun handleFiles(files: Sequence<List<Path>>): Sequence<TestResult> {
+    override fun handleFiles(files: Sequence<TestFiles>): Sequence<TestResult> {
         val fixPluginConfig = testConfig.pluginConfigs.filterIsInstance<FixPluginConfig>().single()
         val generalConfig = testConfig.pluginConfigs.filterIsInstance<GeneralConfig>().single()
 
-        return files.chunked(fixPluginConfig.batchSize!!.toInt()).map { chunk ->
-            val pathMap = chunk.map { it.first() to it.last() }
+        return files.map { it as Test }.chunked(fixPluginConfig.batchSize!!.toInt()).map { chunk ->
+            val pathMap = chunk.map { it.expected to it.test }
```

Comment:
Shouldn't it be vice versa? It was the original problem leading to this change

Reply:
Fixed